packages/manager/modules/sharepoint/package.json

```diff
@@ -23,6 +23,7 @@
     "@ovh-ux/web-universe-components": "^7.0.0",
     "@uirouter/angularjs": "^1.0.15",
     "angular": "^1.7.5",
+    "angular-translate": "^2.18.2",
```

Comment:
No update regarding `yarn.lock` file?

Reply:
@antleblanc It shouldn't be if it's a peer dep, right ?